mappings/net/minecraft/world/level/storage/LevelSummary.mapping

```diff
@@ -29,12 +29,14 @@ CLASS net/minecraft/class_34 net/minecraft/world/level/storage/LevelSummary
 	METHOD method_27430 createDetails ()Lnet/minecraft/class_2561;
 	METHOD method_29586 getVersionInfo ()Lnet/minecraft/class_5315;
 	METHOD method_33405 getConversionWarning ()Lnet/minecraft/class_34$class_5781;
-	METHOD method_33783 isPreWorldHeightChangeVersion ()Z
-		COMMENT Returns whether the level is from a version before the world height was changed to -64 to 320.
+	METHOD method_33783 isPostWorldHeightChangeVersion ()Z
+		COMMENT Returns whether the level is from a version after the world height was changed to -64 to 320.
 		COMMENT
-		COMMENT <p>This includes world versions {@code 2692} and earlier (21w05b and earlier).
+		COMMENT <p>This corresponds to world versions {@code 2693} to {@code 2706}, or
+		COMMENT game versions 21w06a to 21w14a.
 	METHOD method_33784 isUnavailable ()Z
 	METHOD method_35505 getLevelInfo ()Lnet/minecraft/class_1940;
+	METHOD method_38496 isAvailable ()Z
```

Comment:
Maybe mention this is version related like
```suggestion
	METHOD method_38496 isVersionAvailable ()Z
```
Since `isUnavailable` is not a simple inversion of this call